evalai/main.py

```diff
@@ -40,6 +41,7 @@ def main(ctx):
 main.add_command(push)
 main.add_command(set_token)
 main.add_command(submission)
+main.add_command(submission_error)
```

Comment:
How about you nest this under `submission command`:
```
$ evalai submission <ID> stderr`

Reply:
As suggested I updated the changes. please review and let me know in google codein task page.

to test use:

evalai submission 48728 stderr